tasks/init.json

```diff
@@ -0,0 +1,52 @@
+{
+  "description": "Make a HTTP or HTTPS request.",
+  "parameters": {
+    "body": {
+      "description": "The request body.",
+      "type": "Optional[Data]"
+    },
+    "body_format": {
```

Comment:
Opted for `body_format` over `json_endpoint`, as it provides a little more flexibility in indicating if the body should be serialized/parsed as JSON while allowing for other common formats as well.